TYPO3.Flow/Classes/TYPO3/Flow/Security/GlobalObjectStateInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Created by IntelliJ IDEA.
+ * User: sebastian
+ * Date: 17.03.17
+ * Time: 09:58
+ */
+
+namespace TYPO3\Flow\Security;
+
+interface GlobalObjectStateInterface
```

Comment:
agree, I'm only able to fork an add another PR, maybe you can grant me the right - or add the todos directly to the PR description as tasklist?